helpers.sh

```diff
@@ -101,10 +101,13 @@ function get_machine() {
     echo $MACHINE
 
     # If we didn't get a node...
-    if [[ "$MACHINE" != "sh"* ]]
+    if [[ "$MACHINE" != "wheat"* ]]
```

Comment:
This would also work very nicely as a variable.

Reply:
E.g., for both this and the one above, you could have logic in the file to generate the config that if the user selects farmshare, the `$MACHINE_PREFIX` is `wheat`. Otherwise it's `sh`. Then we can avoid adding this logic around the actual scripts and just reference the variable.

Reply:
fixed with boolean operator 
6aa907f
as well as determined a machine prefix in setup.sh